src/app/services/session-storage.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from "@angular/core";
+import AppValues from "../common/app.values";
+
+@Injectable({
+    providedIn: "root",
+})
+export class SessionStorageService {
+    public get sidePanelVisibility(): string | null {
+        return sessionStorage.getItem(AppValues.SESSION_STORAGE_SIDE_PANEL_VISIBLE);
+    }
```

Comment:
Why are you using a getter here?
I think just a "get" method will be enough.

```suggestion
    public getSidePanelVisibility(): string | null {
        return sessionStorage.getItem(AppValues.SESSION_STORAGE_SIDE_PANEL_VISIBLE);
    }
```

But if you have some strong points to use a getter, and it will be used as a parameter, I suggest renaming it to `isSidePanelVisible`

Reply:
Fixed